src/components/Button/Button.tsx

```diff
@@ -6,13 +6,16 @@ import React from "react";
  * @returns styled html button component
  * @author Lia Arroyo <liayzabel@gmail.com>
  */
-const Button = ({ label, onClick }: ButtonInputProps) => {
+const Button = (props: ButtonInputProps) => {
```

Comment:
```suggestion
const Button = ({
  className,
  label,
  disabled,
  widthFull,
  onClick,
}: ButtonInputProps) => {
```
use early destructuring. Makes it look cleaner. Probs need to refactor some stuff below